Formula/minikube.rb

```diff
@@ -0,0 +1,28 @@
+class Minikube < Formula
+  desc "Tool that makes it easy to run Kubernetes locally"
+  homepage "https://kubernetes.io/docs/getting-started-guides/minikube/"
+  url "https://github.com/kubernetes/minikube.git",
```

Comment:
Can we build from source archive? `https://github.com/kubernetes/minikube/archive/v0.26.1.tar.gz`

Reply:
The only problem with that is the make script depends on the build directory having a valid git repository.

Reply:
OK then